partials/html-admin-setup-footer.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * File containing the view used in the footer of the setup pages.
+ *
+ * @package Crowdsignal_Forms\Admin
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+?>
+
+</div>
```

Comment:
ooof, closing tags from all over the world, harsh one

Reply:
I just indented everything and I think fixed the div tags.